### PR TITLE
UI Editor for `light` card

### PR DESCRIPTION
**Pull request by iantrich**
light-editor

### CODE REVIEW

File: src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    if (!this.hass) {
+      return html``;
+    }
+
+    return html`
+      ${configElementStyle}
+      <div class="card-config">
+        <paper-input
+          label="Name"
+          value="${this._name}"
```
Comment (zsarnett) [inline src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts]: Can be `.value` yea?

File: src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { Config } from "../../cards/hui-light-card";
+import { configElementStyle } from "./config-elements-style";
+
+import "../../components/hui-theme-select-editor";
+import "../../components/hui-entity-editor";
+
+const cardConfigStruct = struct({
+  type: "string",
+  id: "string|number",
```
Comment (zsarnett) [inline src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts]: I believe this can be taken out now. @bramkragten

**Review by zsarnett — APPROVED** [review]

File: src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts (hunk 2 above)
Comment (bramkragten) [reply src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts]: It should yeah

Comment (iantrich) [reply src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts]: I'll rebase all my editor PRs and verify

**Pull request merged**